source/documentation/runbooks/dns/rollback_dns_iac.html.md.erb

```diff
@@ -0,0 +1,60 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Rolling back DNS-IAC
+last_reviewed_on: 2022-08-18
+review_in: 3 months
+---
+
+# Rolling back DNS-IAC
+
+> Please be careful when doing this process, if you do not understand Terraform and the AWS DSD R53 estate, seek guidance from someone that does
+
+There can be cases when you wish to rollback any changes made to AWS Route53 using DNS-IAC
+
+This document includes the process for that.
+
+> Be aware this process is new and this page may not cover unexpected steps, please update the process accordingly.
+
+# The process
+
+In order to rollback DNS-IAC the high level process is as follows:
+
+> A laptop authenticated against the DSD AWS account is needed for this process
+
+1. Checkout DNS-IAC
+
+```
+git checkout https://github.com/ministryofjustice/dns-iac.git
+```
+
+2. Checkout the commit you wish to revert to
+
+```
+cd dns-iac && git checkout $commit_hash
+```
+
+3. Run terraform plan on a laptop authenticated against the DSD AWS account
+
+```
+terraform plan
+```
+
+4. Check the changes in Terraform are the ones you wish to revert
+
+> Please be careful when doing this - this can be a very contextual area, you do need to up to date knowledge with planned changes in the Route53 estate
```

Comment:
"you do need to up to date knowledge with planned changes in the Route53 estate" I think there's some words missing here :D